tracer/src/Datadog.Trace/Iast/Iast.cs

```diff
@@ -35,6 +35,15 @@ public Iast()
     private Iast(IastSettings settings = null)
     {
         _settings = settings ?? IastSettings.FromDefaultSources();
+
+        if (_settings.Enabled)
+        {
+            Log.Information("IAST is enabled.");
+        }
+        else
+        {
+            Log.Information("IAST is disabled.");
+        }
```

Comment:
We already record this information in the standard log message (along with extra information) so I don't think we should add this extra log too

```suggestion
```